src/components/Categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <h1>Under construction</h1>
```

Comment:
- [ ] Based on the requirements of this project, this component should display only a button with the text "Check status". Please kindly fix 🔧 it and move a step toward approval 👍